.github/pull_request_template.md

```diff
@@ -16,17 +16,32 @@
 
 ## Types of changes
 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+**Arizona Quickstart** (install profile, custom modules, custom theme)
+- Patch release changes
+   - [ ] Bug fix
+   - [ ] Accessibility, performance, or security improvement
+   - [ ] Critical institutional link or brand change
+- Minor release changes
+   - [ ] New feature
+   - [ ] Breaking or visual change to existing behavior
+   - [ ] Non-critical brand change
+   - [ ] New internal API or API improvement with backwards compatibility
+   - [ ] Risky or disruptive cleanup to comply with coding standards
+   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
 
-## Checklist:
-<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
```

Comment:
Remove this line

Reply:
Looks like this file is the one being used because of this line (missing in the other file, but appears on the template when making a PR)